christfellowshipchurch/web-app-v2
# Removed "clickable" functionality for non-clickable redirect cards

### About
This PR was for the non-clickable redirect cards. Clicking on a non-clickable redirect card caused the scroll to go all the way to the top of the screen. 

### Test Instructions
1. Go to `/trendy-or-timeless#no-click`
2. Scroll to the bottom of the page until you reach the section header "testing horizontal card list feature"
3. Click on the redirect card. No event should take place.

### Screenshots
<img width="1170" alt="Screenshot 2023-06-13 at 5 47 20 PM" src="https://github.com/christfellowshipchurch/web-app-v2/assets/93110503/a87e1518-ff7b-4f7b-912e-f68d64a96484">


### Closes Tickets
[CFDP-2602](https://christfellowshipchurch.atlassian.net/browse/CFDP-2602?atlOrigin=eyJpIjoiYTIwOTFmZGIzODI1NDAyMmEzZmU0MTA0MTdjMmFkNmMiLCJwIjoiaiJ9)



[CFDP-2602]: https://christfellowshipchurch.atlassian.net/browse/CFDP-2602?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
